app/assets/javascripts/application.js

```diff
@@ -9,14 +9,18 @@
 //= require angular-ui-bootstrap
 //= require angular-ui-bootstrap-tpls
 //= require angular-sanitize
+//= require angular-ui-router
+//= require angular-rails4-templates
 //= require moment
 //= require moment-strftime
 //= require moment-timezone
 //= require sprintf
 //= require numeral
 //= require miq_angular_application
+//= require miq_angular_configuration
 //= require services/miq_service
 //= require services/miq_db_backup_service
+//= require component-directives/form_buttons
```

Comment:
I'm thinking we may want to call the directory just `components`, because SSUI already does?